docs/standards/smart-contracts/lsp7-digital-asset.md

```diff
@@ -265,6 +265,206 @@ _Triggers the **[Transfer](#trasnfer-2)** event when tokens get successfully tra
 
 :::
 
+## Internal Functions
+
+:::info Warning
+By deploying the smart contracts directly the methods _burn and _mint will lack implementantion.
```

Comment:
`will lack implementation` is little bit unclear to me, lets use other words.
Maybe:
>  by deploying an LSP7DigitalAsset contract, there will be no public mint or burn function. In order .. extend the smart contract ...